amr-wind/equation_systems/icns/source_terms/CoriolisForcing.cpp

```diff
@@ -69,6 +80,7 @@ void CoriolisForcing::operator()(
     const auto sinphi = m_sinphi;
     const auto cosphi = m_cosphi;
     const auto corfac = m_coriolis_factor;
+    const auto S = m_S;
```

Comment:
if you are trying to use a boolean to take a value of 0 or 1 then create an if statement. I am not sure if this even compiles

Reply:
Is this 100% necessary to include? I can confirm it does compile.

Reply:
Jarred, casting a double or int into a boolean is plain wrong. It will be a great source of memory leakage and I cannot approve that. Please add a double and use the boolean only as the logic to assign the correct value to that double S.

Reply:
I wasn't aware this would be a source of memory leakage. I have added logic to specify double S.